Dockerfile

```diff
@@ -1,16 +1,7 @@
-FROM alpine:3.9
-
-ENV VARNISH_VERSION 6.1
-
-RUN apk add --update --no-cache \
-	bash \
-	varnish \
-	&& rm -rf /var/cache/apk/*
-
-COPY default.vcl /opt/default.vcl
-COPY startup.sh /opt/startup.sh
-COPY healthcheck.sh /opt/healthcheck.sh
+FROM alpine:3.8
```

Comment:
We should use the latest stable alpine version (3.9) unless there are major reasons not to.